Cargo.toml

```diff
@@ -35,7 +35,9 @@ rand = "0.8.4"
 tide = "0.16"
 actix-web = "4"
 tokio = { version = "1", features = ["rt-multi-thread", "net", "macros", "signal"] }
-hyper = { version = "0.14.16", features = ["server", "http1", "tcp"] }
+hyper = { version = "1.3.1", features = ["server", "http1"] }
+hyper-util = { version = "0.1.3", features = ["full"] }
```

Comment:
Are all features required? E.g. the `client` feature?

Reply:
Nice catch! Feature `tokio` is sufficient. Fixed in 043fdd3.